### DISCUSSION TITLE
Add Css Files for Keyframes?

### DISCUSSION THREAD

**Discussion by nvima** [BuilderIO/jsx-lite#973] [Q&A] [answered]
Hey, i want to add some css animation.
Something like this: https://codepen.io/danielmorosan/pen/XmYBVx
if have tried to implement the css inline with css={{}}, but some keys are generated wrong ex. -webkit-animation
Also to include the keyframes I think you need a css file. I think this is not possible with css={{}} ?
`@-webkit-keyframes sk-bouncedelay {
  0%, 80%, 100% { -webkit-transform: scale(0) }
  40% { -webkit-transform: scale(1.0) }
}`

I found out that papanasi uses css files in there components like in this repo dir:
https://github.com/CKGrafico/papanasi/tree/main/src/elements/components/avatar

I've tried to import a css file, but the css file gets not generated. I tried to find out how they do it, but couldn't find anything. Is there a simple solution to include css animations with keyframes with mitosis?

**Comment by nvima** [comment]
i've found a little workaround for my usecase. 
svg has animation tags.
https://codepen.io/nvima/pen/VwBPYRx

**Comment by samijaber** [comment]
> if have tried to implement the css inline with css={{}}, but some keys are generated wrong ex. -webkit-animation

Please file a github issue using the bug template, so we can further investigate these bugs.

**Comment by samijaber** [comment, marked as answer]
> I've tried to import a css file, but the css file gets not generated. I tried to find out how they do it, but couldn't find anything. Is there a simple solution to include css animations with keyframes with mitosis?

If you want to import a CSS file, you're going to have to add the infrastructure necessary for each individual output, to process those CSS files using whatever methods you want (e.g. PostCSS plugin, CSS Modules, SASS, etc.)

Mitosis itself does not do anything to handle CSS files for you, as it only concerns itself with compiling the component file.

**Reply by samijaber** [reply]
Just saw Papanasi's setup: they are using Rollup for bundling, and therefore added a Rollup PostCSS plugin:

https://github.com/CKGrafico/papanasi/blob/56ed9dd3c670f9b7588e356a7ff8a53e096f8082/rollup.config.js#L64
https://github.com/CKGrafico/papanasi/blob/56ed9dd3c670f9b7588e356a7ff8a53e096f8082/postcss.config.js
